modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view_file_entry.jsp

```diff
@@ -41,7 +41,7 @@ if (portletTitleBasedNavigation) {
 }
 %>
 
-<div class="<%= portletTitleBasedNavigation ? StringPool.BLANK : "closed sidenav-container sidenav-right" %>" id="<%= liferayPortletResponse.getNamespace() + (portletTitleBasedNavigation ? "FileEntry" : "infoPanelId") %>">
+<div class="<%= portletTitleBasedNavigation ? StringPool.BLANK : "closed sidenav-container sidenav-right" %>" id="<%= liferayPortletResponse.getNamespace() + (portletTitleBasedNavigation ? "FileEntry" : ("infoPanelId" + StringPool.UNDERLINE + fileEntry.getFileEntryId())) %>">
```

Comment:
I like this idea, but the `div`'s `id` and `liferay-frontend:management-bar-sidenav-toggler-button`'s `sidenavId` are not the same.
The `div`'s `id` depends on whether `portletTitleBasedNavigation` is `true` or not
`id="<%= liferayPortletResponse.getNamespace() + (portletTitleBasedNavigation ? "FileEntry" : ("infoPanelId" + StringPool.UNDERLINE + fileEntry.getFileEntryId())) %>"`

Reply:
@IstvanD They will be the same, because `<liferay-frontend:management-bar-sidenav-toggler-button>` is wrapped in `<c:if test="<%= !portletTitleBasedNavigation %>">`, see [here](https://github.com/liferay-frontend/liferay-portal/blob/aac18473e9a0dc7258d2bd3604a0f38e318fe5d1/modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view_file_entry.jsp#L99). It is not rendered if `portletTitleBasedNavigation` is true.